bot/modules/lbrylink.js

```diff
@@ -12,31 +12,52 @@ exports.lbrylink = function(bot, msg, suffix) {
     var link = msg.content.includes("lbry://");
     if (link) {
       var text = msg.content.replace("lbry://", "https://open.lbry.io/");
-      var message = text.match(/\bhttps?:\/\/\S+/gi);
+      var message = text.match(/\bhttps?:\/\/\S+/gi).toString();
+      var check1 = message.includes(">");
       if (ResponseDebug == "true") {
```

Comment:
and here you can do `if (ResponseDebug)`

(also it's good practice to begin variables with small case)